### PR TITLE
Feature/5

### PR DESCRIPTION
**Pull request by akuraya1**
I watched lesson 5 but used a more convenient method for me I used cellspacing="1" cellpadding="15" border="1" because in my opinion it is more convenient for me than typing in each line
rowspan or colspan + this table does not need to use them

### CODE REVIEW

File: lesson-5/index.html [hunk 1]
```diff
@@ -3,9 +3,48 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
+    <link rel="icon" type="image/x-icon" href="file:///D:/images/1.png">
```
Comment (RomanIlkiv) [inline lesson-5/index.html:7]: same mistake with link to image

File: lesson-5/index.html [hunk 2]
```diff
@@ -3,9 +3,48 @@
...
 </head>
 <body>
-    
+    <h1>Artem Nadolskij :3</h1>
+    <img alt="human" width="300" height="315" src="file:///D:/images/photo_2023-08-04_15-12-18.jpg"><br>
```
Comment (RomanIlkiv) [inline lesson-5/index.html:11]: here the same

File: lesson-5/index.html [hunk 3]
```diff
@@ -3,9 +3,48 @@
...
+    <h2>Senior Front-End Developer</h2><br>
+    <h3><ins>Work Expience .</ins></h3>
+    <table cellspacing="1" cellpadding="15" border="1">
+        <tr>
+            <thead>
+            <th><i><b>Period</b></th>
+                <th><i><b>Position</b></th>
```
Comment (RomanIlkiv) [inline lesson-5/index.html:18]: bad formatting

File: lesson-5/index.html [hunk 4]
```diff
@@ -3,9 +3,48 @@
...
+            <td><i><b>2013-2015</b></td>
+                <td><i><b>Senior Frontend Developer Work In<a href="https://Google.com">Google.com</a></b></td>         
+        </tr>
+        <tr>
+            <td><i><b>2015-17</b></td>
+                <td><i><b>Senior Frontend Developer Work In<a href="https://Youtube.com">Youtube.com</a></b></td>         
+        </tr>
+        <tr>
+            <td><i><b>2017-2019</b></td>
+                <td><i><b>Senior Frontend Developer Work In<a href="https://Facebook.com">Facebook.com</a></b></td>
```
Comment (RomanIlkiv) [inline lesson-5/index.html:31]: bad formatting
why yo add extra space before second `td`?

Comment (akuraya1) [reply lesson-5/index.html]: https://github.com/akuraya1/r_b/pull/6/commits/7fda5b96a731ca6f0454217f45450afef7cd8030  moved all the extra stuff so it's better or what's wrong?

**Pull request merged**